Makefile

```diff
@@ -0,0 +1,50 @@
+VGO=go
+GOFILES := $(shell find cmd internal pkg -name '*.go' -print)
+GOBIN := $(shell $(VGO) env GOPATH)/bin
+LINT := $(GOBIN)/golangci-lint
+MOCKERY := $(GOBIN)/mockery
+
+# Expect that FireFly compiles with CGO disabled
+CGO_ENABLED=0
+GOGC=30
+
+.DELETE_ON_ERROR:
+
+all: build test go-mod-tidy
+test: deps lint
+		$(VGO) test ./internal/... ./cmd/... ./pkg/... -cover -coverprofile=coverage.txt -covermode=atomic -timeout=30s
+coverage.html:
+		$(VGO) tool cover -html=coverage.txt
+coverage: test coverage.html
+lint: ${LINT}
+		GOGC=20 $(LINT) run -v --timeout 5m
+${MOCKERY}:
+		$(VGO) install github.com/vektra/mockery/cmd/mockery@latest
+${LINT}:
+		$(VGO) install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+
+
+define makemock
+mocks: mocks-$(strip $(1))-$(strip $(2))
+mocks-$(strip $(1))-$(strip $(2)): ${MOCKERY}
+	${MOCKERY} --case underscore --dir $(1) --name $(2) --outpkg $(3) --output mocks/$(strip $(3))
+endef
+
+$(eval $(call makemock, pkg/ethsigner,       Wallet,  ethsignermocks))
+$(eval $(call makemock, internal/rpcserver,  Server,  rpcservermocks))
+$(eval $(call makemock, internal/rpcbackend, Backend, rpcbackendmocks))
+
+firefly-signer: ${GOFILES}
+		$(VGO) build -o ./firefly-signer -ldflags "-X main.buildDate=`date -u +\"%Y-%m-%dT%H:%M:%SZ\"` -X main.buildVersion=$(BUILD_VERSION)" -tags=prod -tags=prod -v ./ffsigner 
+go-mod-tidy: .ALWAYS
+		$(VGO) mod tidy
+build: firefly-signer
+.ALWAYS: ;
+clean:
+		$(VGO) clean
+deps:
+		$(VGO) get ./ffsigner
+swagger:
+		$(VGO) test ./internal/apiserver -timeout=10s -tags swagger
```

Comment:
Does this repo have an API server/Swagger reference?

Reply:
No, everything is just the JSON/RPC `POST` `/` endpoint. Will remove this from `Makefile`